services/managers/srp_manager.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+
+
+import json
+import urllib2
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+class srpManager():
+    @staticmethod
+    def get_kill_id (killboard_link):
+        str = (killboard_link)
+        set = '0123456789'

```

Comment:
`set` is a protected name (a data type), I'm surprise this isn't error-ing out.

Did you look into my regex suggestion?